deaf-and-hard-of-hearing/event-organizers/conference-best-practices.md

```diff
@@ -0,0 +1,151 @@
+
+Have some reserved seating at the front for deaf and hard of hearing attendees. Many rely on lip-reading, so you want to be sure they can see the speakers well. And please ask speakers not to wear masks while presenting and provide social distancing opportunities instead. 
+
+### Sign language interpreters
+
+Provide interpreters upon request and in advance. We also recommend having two sign language interpreters on-call to accommodate last-minute registrants or miscommunications. Ask interpreter agencies about their cancellation policies — these can vary significantly. A week's notice, 48, or even 24 hours with a fee, are all possible, depending on the agency. 
+
+Obviously, a big conference like KubeCon has more resources than a local community event — we get that. Since we also want to be included in smaller events, we'll offer various options. To be clear, the more cost-effective options aren't as inclusive, meaning we'll miss out on many opportunities our hearing peers benefit from. Do only go the more cost-effective route if the alternative is no accessibility. We deserve to be included and appreciate your support. 
+
+Sign language interpreters are the most valuable but also the most expensive accommodation. Incorporate the cost into the overall conference cost. It should be no different than including audio/video equipment, swag, food, refreshments, etc. into your budget. 
+
+#### Include interpreters in recording
+
+If you record your sessions, be sure the interpreters are on screen so deaf attendees can benefit from your accessibility effort even when watching them on-demand. 
+
+#### Global or national conferences 
+
+If you only have a few deaf individuals attending, provide two interpreters per individual (they always come in tandem for engagements longer than one hour). Wherever the deaf person goes, they'll follow. This will allow deaf individuals not only to attend sessions, but also network with their peers, walk the showfloor and talk with vendors. This is particularly important if they are the only or one of a few deaf people attending, as any interaction is contingent on the availability of an interpreter.
+
+Of course, this approach doesn't scale well. If you have more deaf attendees, you'll need to share accommodations among more individuals. 
+
+Before the conference, ask attendees to sign up for sessions in advance so you know which sessions must be staffed by interpreters. But allow for last-minute changes and be prepared to inform interpreters about these. While you'll want to plan ahead, you also want to offer attendees the flexibility to change their minds. Maybe they met someone who's giving a talk, and they'd like to attend that session. If you have multiple deaf attendees, have interpreters at keynotes by default. 
+
+##### Accommodation desk
+
+Set up an accommodation desk and/or communication channel where attendees can ask questions, request changes, or get an interpreter spontaneously, if available. 
```

Comment:
Set up an accommodation desk and/or communication channels where attendees can ask questions, request changes, or get an interpreter spontaneously, if available. **Be sure to staff it with people familiar with deaf/hoh needs and have assistive technology available to assist with communication when needed.**